script/tool/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: flutter_plugin_tools
 description: Productivity and CI utils for flutter/packages
 repository: https://github.com/flutter/packages/tree/main/script/tool
-version: 0.13.4+3
```

Comment:
You can revert the version change; we no longer publish this as a package, it just hasn't been updated for that change yet.